### PR TITLE
iOS and Android image segmentation tutorials; minor recipes code and typo fix for 1.7

### PR DESCRIPTION
**Pull request by jeffxtang**

### CODE REVIEW

**Comment by chauhang** [comment]
@jeffxtang  Please split into 2 separate PRs. One for the typos fixes and minor edits, and other for the new content

File: beginner_source/deeplabv3_on_android.rst [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+2. Get example input and output of the model in Python for Android app to compare with;
+
+3. Build a new Android app, or reuse an Android example app, to load the converted model;
+
+4. Prepare the model input that can be accepted by the model and process the model output;
+
+5. Complete the UI, refactor, build and run the app to see image segmentation in action.
+
+Pre-requisites
```
Comment (jeremiahschung) [inline beginner_source/deeplabv3_on_android.rst:30]:
```suggestion
Prerequisites
```

File: beginner_source/deeplabv3_on_ios.rst [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+2. Get example input and output of the model in Python for iOS app to compare with;
+
+3. Build a new iOS app, or reuse an iOS example app, to load the converted model;
+
+4. Prepare the model input that can be accepted by the model and process the model output;
+
+5. Complete the UI, refactor, build and run the app to see image segmentation in action.
+
+Pre-requisites
```
Comment (jeremiahschung) [inline beginner_source/deeplabv3_on_ios.rst:30]:
```suggestion
Prerequisites
```

File: beginner_source/deeplabv3_on_ios.rst [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+---------------
+
+* PyTorch 1.6 or 1.7
+
+* torchvision 0.7 or 0.8
+
+* Xcode 11 or 12
```
Comment (jeremiahschung) [inline beginner_source/deeplabv3_on_ios.rst:37]: Let's use "version X or later" instead. (e.g. PyTorch 1.6 or later)

File: beginner_source/deeplabv3_on_android.rst [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+---------------
+
+* PyTorch 1.6 or 1.7
```
Comment (jeremiahschung) [inline beginner_source/deeplabv3_on_android.rst:33]: Let's use "version X or later" instead. (e.g. PyTorch 1.6 or later)

**Pull request merged**